spec/queries/my_facilities_query_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe MyFacilitiesQuery do
+  context 'utility methods' do
```

Comment:
We can remove this context block. I don't think it adds too much useful info to the spec.